src/Parser/Hunter/Marksmanship/Modules/Talents/LockAndLoad.js

```diff
@@ -81,14 +84,77 @@ class LockAndLoad extends Analyzer {
     this.autoShots += 1;
   }
 
+  get expectedProcs() {
+    return this.autoShots * PROC_CHANCE;
+  }
+
+  GetZPercent(z) {
+    // If z is greater than 6.5 standard deviations from the mean
+    // the number of significant digits will be outside of a reasonable
+    // range.
+    if (z < -6.5)
+      return 0.0;
+
+    if (z > 6.5)
+      return 1.0;
+
+    let factK = 1;
+    let sum = 0;
+    let term = 1;
+    let k = 0;
+    const loopStop = Math.exp(-23);
+
+    while (Math.abs(term) > loopStop) {
+      term = 0.3989422804 * Math.pow(-1, k) * Math.pow(z, k) / (2 * k + 1) /
+        Math.pow(2, k) * Math.pow(z, k + 1) / factK;
```

Comment:
https://i.imgur.com/TrymOlU.png

Reply:
I got help from stackoverflow 😅 but verified it works